lib/services/insightsManagement/package.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
Please update the content of License.txt in this folder to that of MIT license. You can copy this content from [here](https://raw.githubusercontent.com/Azure/azure-sdk-for-node/master/lib/services/storageManagement2/LICENSE.txt)